dwitter/templates/base.html

```diff
@@ -37,23 +37,24 @@
       // Check for compatibility
       if (window.localStorage){
         var switchTheme = function(){
-          // Change the imported stylesheet from main.css to main_dark.css or vice versa
-          // Change localstorage theme to 'dark'
+          // Shift themes
           var theme = localStorage.getItem('theme');
-          if (theme == 'dark') {
-            localStorage.setItem('theme','');
+          if (theme == 'default') {
+            localStorage.setItem('theme','black');
+          } else if (theme == 'black') {
+            localStorage.setItem('theme','green');
           } else {
-            localStorage.setItem('theme','dark');
+            localStorage.setItem('theme','default');
           }
           location.reload();
           // Call this function in the li to change the mode
         }
 
         // Do the same thing outside the function without changing localStorage to set correct theme
         var theme = localStorage.getItem('theme');
-        if (theme == 'dark'){
+        if (theme != 'default'){
```

Comment:
We should probably make this

```js
if (theme && theme !== 'default') {
```

because I believe `''` is what new users get to start with. (Making `''` and `'default'` the same thing.)

Reply:
That's actually a very good idea!

Reply:
I think you may have edited the wrong bit of code. 🍭